packages/table/src/Table.ts

```diff
@@ -507,6 +507,11 @@ export class Table extends SizedMixin(SpectrumElement, {
                 }
             );
         }
+
+        // Ensures screenreaders can announce the true size of the table
+        // despite virtualization only rendering a subset of rows.
+        this.setAttribute('aria-rowcount', `${this.items.length}`);
```

Comment:
Because this code path is only run when `items` has a `length` were this table to be used with filtering or searching and the length could conceivable become `0`, there is a change that this data would be out of date. Should we remove this attribute in the implied `else` at line 474?